core-services/prow/02_config/_config.yaml

```diff
@@ -412,6 +412,9 @@ jenkins_operators:
   max_concurrency: 1
   max_goroutines: 1
 log_level: debug
+managed_webhooks:
```

Comment:
Looks like default values.
Can we try to remove those?

Reply:
Yeah. They were added by `make update`, but I can take it out.

Reply:
@hongkailiu FYI, after I took those lines out and pushed the new commit, I ran `make update` again and it added them back.